core/src/genesis.rs

```diff
@@ -395,9 +405,19 @@ impl RawGenesisDomainBuilder {
 
     /// Add an account to this domain
     #[must_use]
-    pub fn with_account(mut self, account_name: Name, public_key: PublicKey) -> Self {
+    pub fn with_account(self, account_name: Name, public_key: PublicKey) -> Self {
+        self.with_account_meta(account_name, public_key, Default::default())
+    }
+
+    pub fn with_account_meta(
```

Comment:
```suggestion
    pub fn account_with_meta(
```

I would name it like this, the previous name make me think we're adding metadata to an existing account instead of registering an account with metadata in one instruction